### ISSUE TITLE
OpenTK 3.1 crashes on startup on 64+ button joysticks

### ISSUE THREAD

**Issue by tyronx** [opentk/opentk#1020]
### Description

OpenTK just straight up crashes if any joystick is connected to the device that has more than 64 buttons

### Repro steps

1. Install vJoy: http://vjoystick.sourceforge.net/site/index.php/download-a-install/download
2. Configure vJoy, set one Joystick to 70 buttons
3. Launch any application that uses OpenTK 3.1. Crashes with below exception

```
System.ArgumentOutOfRangeException: Das angegebene Argument liegt außerhalb des gültigen Wertebereichs.
Parametername: button
   bei OpenTK.Input.JoystickState.SetButton(Int32 button, Boolean value)
   bei OpenTK.Platform.Windows.WinRawJoystick.QueryDeviceCaps(Device stick)
   bei OpenTK.Platform.Windows.WinRawJoystick.RefreshDevices()
   bei OpenTK.Platform.Windows.WinRawJoystick..ctor(IntPtr window)
   bei OpenTK.Platform.Windows.WinRawInput.CreateDrivers()
   bei OpenTK.Platform.Windows.WinInputBase.ProcessEvents()
   bei System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   bei System.Threading.ThreadHelper.ThreadStart()
-------------------------------
```

There's apparently a hardcoded array of 64 buttons, once it goes beyond that the init code crashes. 

I already brute-force fixed this for my needs by just catching exceptions during joystick init and then ignoring that joystick, diff here: https://github.com/anegostudios/opentk/commit/c40eabd87385fbbbf7ebe5b8029c2f0a5b88a222

A build of OpenTk with that fix in there will roll out to a sizable amount of users within a few days so that should sufficiently test the change.

I can make a merge request to the 3.x branch if you like.

**Comment by peppy** [comment]
If you need a quick solution, here's our patch for it: https://github.com/ppy/osuTK/pull/59

**Comment by varon** [comment]
@tyronx - Must have missed this.

We'd love to merge a fix for this.

**Comment by tyronx** [comment]
@varon mind if I do the PR together with some build script fixes for windows?

**Comment by varon** [comment]
@tyronx  - That would be fantastic.

**Comment by tyronx** [comment]
This has been fixed in released in OpenTK 3.1, so closing issue.
